src/main/java/io/github/wimdeblauwe/hsbt/mvc/HtmxMvcConfiguration.java

```diff
@@ -1,34 +1,35 @@
 package io.github.wimdeblauwe.hsbt.mvc;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.List;
+
 import org.springframework.beans.factory.ObjectFactory;
 import org.springframework.boot.autoconfigure.AutoConfiguration;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
 import org.springframework.boot.autoconfigure.web.servlet.WebMvcRegistrations;
+import org.springframework.context.ApplicationContext;
 import org.springframework.util.Assert;
 import org.springframework.web.method.support.HandlerMethodArgumentResolver;
 import org.springframework.web.servlet.LocaleResolver;
 import org.springframework.web.servlet.ViewResolver;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
-import org.thymeleaf.spring6.view.ThymeleafViewResolver;
 
-import java.util.List;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 @AutoConfiguration
 @ConditionalOnWebApplication
 public class HtmxMvcConfiguration implements WebMvcRegistrations, WebMvcConfigurer {
 
-    private final ViewResolver resolver;
+    private final ApplicationContext context;
     private final ObjectFactory<LocaleResolver> locales;
     private final ObjectMapper objectMapper;
 
-    HtmxMvcConfiguration(ThymeleafViewResolver resolver, ObjectFactory<LocaleResolver> locales, ObjectMapper objectMapper) {
-        Assert.notNull(resolver, "ViewResovler must not be null!");
+    HtmxMvcConfiguration(ApplicationContext context, ObjectFactory<LocaleResolver> locales, ObjectMapper objectMapper) {
```

Comment:
Would it be possible to inject `@Qualifier("viewResolver")ViewResolver viewResolver` instead of the whole context?

Reply:
I don't think the `ViewResolver` is available so early. I tried it and got an exception, so I moved the resolution out of the constructor and it worked.